server/src/api/schemas.py

```diff
@@ -64,6 +64,30 @@ class ActionIn(Schema):
     action = fields.String(required=True, validate=OneOf(["cancel"]))
 
 
+class Attachment(Schema):
+    """Attachment pathnames schema
+
+    - `local`: path to attachment on the runner
+    - `agent`: path to copy the attachment in the testflinger agent (optional)
+    - `device`: path to copy the attachment in the device under test (optional)
```

Comment:
What's the difference between "local" (the "runner") and "agent" here?

Reply:
The local/runner filenames refer to files on the machine where the job is submitted. That's where the attachments are collected and packed. That could be on the machine running the testflinger CLI or a Github runner. You may want to rename/move these files on the testflinger agent, so then you can specify their new name and location (always under `attachments/{phase}` with the `agent` field.